.editorconfig

```diff
@@ -0,0 +1,11 @@
+# indicate this is the root of the project
+root = true
+
+[*.{kt,java,xml,gradle,md}]
+charset = utf-8
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
+insert_final_newline = true
+end_of_line = lf
+disabled_rules=no-wildcard-imports
```

Comment:
Why have you disabled this? Should not be disabled

Reply:
Otherwise I couldn't solve the error in ktlint "Wildcard import (cannot be auto-corrected)"

Reply:
https://detekt.dev/style.html#magicnumber this explains all the errors meanings

Reply:
In order to solve the issue you have to remove the importing of all the code (".*") and substitute it with the things that you actually use